lang/solidity/files/patch-libsolutil_JSON.cpp.diff

```diff
@@ -0,0 +1,15 @@
+--- libsolutil/JSON.cpp.orig	2023-03-15 21:14:24.000000000 +0800
++++ libsolutil/JSON.cpp	2023-03-15 21:14:48.000000000 +0800
+@@ -32,10 +32,12 @@
+ 
+ using namespace std;
+ 
++/*
+ static_assert(
+ 	(JSONCPP_VERSION_MAJOR == 1) && (JSONCPP_VERSION_MINOR == 9) && (JSONCPP_VERSION_PATCH == 3),
+ 	"Unexpected jsoncpp version: " JSONCPP_VERSION_STRING ". Expecting 1.9.3."
+ );
++*/
```

Comment:
Have you filed a bug report with upstream to discuss why they mandate this specific version of jsoncpp and what consequences we might expect from bypassing that requirement?